src/main/java/io/temporal/internal/testservice/TestWorkflowMutableStateImpl.java

```diff
@@ -1390,7 +1401,7 @@ public void startWorkflow(
               scheduleWorkflowTask(ctx);
             }
 
-            int runTimeoutSeconds = startRequest.getWorkflowRunTimeoutSeconds();
+            long runTimeoutSeconds = Durations.toSeconds(startRequest.getWorkflowRunTimeout());
```

Comment:
java duration here and for the other variables in this file?

Reply:
If it improves readability